### DISCUSSION TITLE
How merge/fusion transformer files .safetensors to a whole .safetensors checkpoint ?

### DISCUSSION THREAD

**Discussion by BenDes21** [lllyasviel/stable-diffusion-webui-forge#1578] [Q&A]
Hi there, I got diffusion_pytorch_model-00001-of-00003.safetensors, diffusion_pytorch_model-00002-of-00003.safetensors and diffusion_pytorch_model-00003-of-00003.safetensors after full finetuned Flux with my dataset, I guess at this point I will have to fusion/merge theses 3 parts for created the whole checkpoint usable for Forge . Is anyone know a script or a tool who can allow this last step ? 

**Thanks a lot**

**Comment by kerizane** [comment]
https://github.com/NotTheStallion/reshard-safetensors
